packages/frontend/editor-ui/src/app/components/MainHeader/MainHeader.vue

```diff
@@ -178,13 +175,17 @@ function onTabSelected(tab: MAIN_HEADER_TABS, event: MouseEvent) {
 
 async function navigateToWorkflowView(openInNewTab: boolean) {
```

Comment:
Two issues to note here, when navigating between a saved workflow and a new workflow (using back button)
1. no modal asking to saving workflow
2. canvas position is not reset.

https://github.com/user-attachments/assets/69054c9c-dbf9-4dfd-b155-e228b6057336